README.md

```diff
@@ -1,2 +1,88 @@
-# robotec-generic-assets
-Repository for all domain agnostic O3DE assets
+# Repository content
+
+This repository contains O3DE gems with assets that are not specific to any of the following areas:
+- mining
+- warehouse
+- agriculture
+
+This means, that assets with objects which are designed and primarily used in one of these areas must be located in the area-specific repository, e.g. tractor must be located in the agriculture repository, even though it may be occasionally used in different applications. Objects that do not meet this criterion must be located here, e.g. pickup trucks, even though it is quite common in agriculture.
+
+# Type of assets
+
+This repository will host asset gems, containing all types of O3DE objects except for source code. Permitted assets include:
+- prefabs
+- meshes
+- materials and textures
+- HDRI maps (*.exr)
+- LUA and script canvas scripts
+- URDF files
+
+# Repository structure
+
+All the content is divided into O3DE gems. Each gem will contain a set of substantively related assets. The content of these sets must be as narrow as possible. It is preferred to have a few specific gems, e.g. "road signs", "road guardrails", etc. instead of one generic, e.g. "road infrastructure".
+
+This repository contains two types of gems:
+
+## Gems with "raw" assets
+
+The main purpose of these gems is to provide models, materials, and textures (including HDRi maps). If it is applicable, prefabs should be included integrating other objects of asset. The purpose of these prefabs is to facilitate inserting assets on the scene. Such prefabs however must not contain any functionality beyond the absolute minimum necessary to show assets correctly. Prefabs must not contain in particular:
+- Physics (rigid bodies, joints, etc.)
+- ROS components
+- Scripts
+
+Raw-type gems may depend only on official gems distributed with the O3DE engine. These dependencies should be reduced to a minimum. Raw-type gems must not depend on any external gems, including those contained in o3de-extras repository.
+
+## Gems with "working" assets
+
+The purpose of these gems is to provide prefabs prepared to be used in different applications. These prefabs may be built around "raw" assets (hosted in raw-type gems) and should be extended by additional functionality. This may include for example:
+- Adding physics structure (rigid bodies, joints, etc.)
+- Adding ROS components
+- Adding scripts
+- Joining multiple prefabs (from raw-type or working-type gems)
+
+Working-type gems must not contain models, materials, or textures. Assets from raw-type gems should be used instead.
+
```

Comment:
Working-type gem should not require any other gem other than :
- "Atom",
- "PrimitiveAssets",
- "AtomContent",
- "DiffuseProbeGrid",
- "DevTextures",
- "EMotionFX",
- "FastNoise",
- "GradientSignal",
- "Terrain",
- "Vegetation", 
- "WhiteBox"

Reply:
IMO we shouldn't make such limitations. There are some other gems, I can imagine might be used (weather, clouds, vehicle dynamics, not to mention ROS2). And I believe, more will follow